components/eamxx/src/control/atmosphere_driver.cpp

```diff
@@ -415,12 +415,20 @@ void AtmosphereDriver::setup_column_conservation_checks ()
 }
 
 void AtmosphereDriver::add_additional_column_data_to_property_checks () {
+  // Get list of additional data fields from driver_options parameters.
+  // If no fields given, return.
+  using vos_t = std::vector<std::string>;
+  auto additional_data_fields = m_atm_params.sublist("driver_options").get<vos_t>("property_check_data_fields",
+                                                                                  {"NONE"});
+  if (additional_data_fields == vos_t{"NONE"}) return;
+
+  // Add requested fields to property checks
   auto phys_field_mgr = m_field_mgrs[m_grids_manager->get_grid("Physics")->name()];
-  if (phys_field_mgr->has_field("landfrac")) {
-    m_atm_process_group->add_additional_data_fields_to_property_checks(phys_field_mgr->get_field("landfrac"));
-  }
-  if (phys_field_mgr->has_field("phis")) {
-    m_atm_process_group->add_additional_data_fields_to_property_checks(phys_field_mgr->get_field("phis"));
+  for (auto fname : additional_data_fields) {
```

Comment:
So we're hard-coding the fact that this feature is only available for phys grid checks, right? I don't think there's anything wrong with that (all atm procs _must_ remap the computed fields back to that grid), but maybe we should put an inline comment, to recall that. Maybe even in the xml defaults I would add something like `(only for physics grid checks)` to the doc string...

Reply:
Added a comment. Also, if the user requests a field not on phys grid it will error out and in the error message mention that it is not found on the phys grid.